lms/static/sass/views/_login-register.scss

```diff
@@ -3,6 +3,17 @@
 @import '../base/grid-settings';
 @import "neat/neat"; // lib - Neat
 
+body.view-student-account-access {
```

Comment:
We've actually decided to punt this to ENT-368 so we can think about the implications a bit more, so you can remove this.